go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1671,6 +1676,18 @@ func newNodePortWatcher(patchPort, gwBridge, gwIntf, nodeName string, ips []*net
 		}
 	}
 
+	// ORDER IS important call drop after service forward (since iptables are appended)
+	if config.Gateway.DisableForwardingOnInterfaces {
```

Comment:
when `DisableForwardingOnInterfaces` is false don't we still want to add the drop rule you initially proposed?
-A FORWARD -i breth0 -o breth0 -j DROP
Otherwise the issue with looping traffic will still be there when `DisableForwardingOnInterfaces` is not set

Reply:
So upstream we don't plan to fix that bug. We are going to fix it downstream where DisableForwardInterfaces is going to be true (metalLB OCP use cases).